torch/distributed/pipelining/PipelineSchedule.py

```diff
@@ -602,6 +607,101 @@ def step(self, *args, target=None, losses: Optional[List] = None, **kwargs):
         # Does not contain the last stage
         return None
 
+    def _step_microbatches(
+        self,
+        arg_mbs: Optional[List] = None,
+        kwarg_mbs: Optional[List] = None,
+        target_mbs: Optional[List] = None,
+        losses: Optional[List] = None,
+    ):
+        """
+        Operate on the microbatches using the interleaved 1f1b schedule.
+
+        TODO: Does not use sorted_batch_isend_irecv(). As a result, this schedule does
```

Comment:
can you file an issue for this? 
 - for either adding this support back later if we want it
 - or adding some check that the stage passed here does not have skip-connections

Reply:
Thanks, added here to track: https://github.com/pytorch/pytorch/issues/127964